persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java

```diff
@@ -688,6 +690,12 @@ public T execute(Object... args) throws Exception {
                             return null;
                         }
                     }
+                } catch (ElasticsearchStatusException ese) {
+                    if (ese.status().equals(RestStatus.NOT_FOUND)) {
+                        // this can happen if we are just testing the existence of the item, it is not always an error.
+                        return null;
+                    }
+                    throw new Exception("Error loading itemType=" + clazz.getName() + " itemId=" + itemId, ese);
```

Comment:
```suggestion
                    throw new Exception("Error loading itemType=" + class.getName() + " itemId=" + itemId, ese);
```

Reply:
I'm sorry but this suggestion is not correct because clazz is an argument to the method, not a typo :)